notebooks/name-entity-extraction.ipynb

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
<p>"the following function can build our app:"</p><p> </p><p>replace with:</p><p> </p><p>"The <code>app_for_pipeline</code> function will build a small app which includes a text prompt and will render the entities which were identified."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/optimum-graphcore-fork/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>